src/policy/copyspace.rs

```diff
@@ -125,6 +125,15 @@ impl<VM: VMBinding> CopySpace<VM> {
 
     pub fn prepare(&self, from_space: bool) {
         self.from_space.store(from_space, Ordering::SeqCst);
+        if let MetadataSpec::OnSide(side_forwarding_status_table) =
+            *<VM::VMObjectModel as ObjectModel<VM>>::LOCAL_FORWARDING_BITS_SPEC
+        {
+            side_metadata::bzero_metadata(
```

Comment:
This is performance-critical. We may switch to some sort of "circular" forwarding status in the future to avoid this bulk zeroing.

Reply:
You can put this as TODO in the code. Also probably make it clear with comments that this zeroing is only needed for side metadata, as for inheader forwarding bits, the forwarded objects are dead and we do not need to clear their metadata.